packages/melange/melange.2.0.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Toolchain to produce JS from Reason/OCaml"
+maintainer: ["Antonio Nuno Monteiro <anmonteiro@gmail.com>"]
+authors: ["Antonio Nuno Monteiro <anmonteiro@gmail.com>"]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/melange-re/melange"
+bug-reports: "https://github.com/melange-re/melange/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "5.1"}
```

Comment:
Is this strict bound correct?

Reply:
Yes, we’re dropping support for older versions of OCaml in v2, and making melange only work on 5.1 for now